appveyor.yml

```diff
@@ -5,6 +5,10 @@ image:
 configuration: Release
 clone_depth: 50
 
+environment:
+  api_key:
+    secure: EjcBgIRpoy3pxVyKUnQFNg==

```

Comment:
Here should be `uLmHVukKUWd0jiGGi3V1V9eomr8hCLB75dYvr1BkO0piqqIkuIF+fB/gwVWJ0zYQ`

Reply:
I have changed nuget api key